packages/garbo/src/familiar/freeFightFamiliar.ts

```diff
@@ -68,6 +72,27 @@ export function menu(options: MenuOptions = {}): GeneralFamiliar[] {
       });
     }
 
+    if (
+      gooseDroneEligible() &&
+      get("gooseDronesRemaining") < copyTargetCount()
+    ) {
+      familiarMenu.push({
+        familiar: $familiar`Grey Goose`,
+        expectedValue:
+          mode === "free"
+            ? // It takes 9 experience to go from level 5 to 6 and emit a drone
+              clamp(
+                getModifier("Familiar Experience") / 9,
+                0,
+                // The limit to how valuable any emission can be is how many drones are actually gonna hit the copyTarget
+                copyTargetCount() - get("gooseDronesRemaining"),
+              ) * valueDrops(globalOptions.target)
+            : 0,
+        leprechaunMultiplier: 0,
+        limit: "experience",
+      });
+    }
+
```

Comment:
```suggestion
    if (
      mode === "free" &&
      gooseDroneEligible() &&
      get("gooseDronesRemaining") < copyTargetCount()
    ) {
      familiarMenu.push({
        familiar: $familiar`Grey Goose`,
        expectedValue:
            // It takes 9 experience to go from level 5 to 6 and emit a drone
            clamp(
              getModifier("Familiar Experience") / 9,
              0,
              // The limit to how valuable any emission can be is how many drones are actually gonna hit the copyTarget
              copyTargetCount() - get("gooseDronesRemaining"),
            ) * valueDrops(globalOptions.target),
        leprechaunMultiplier: 0,
        limit: "experience",
      });
    }
```

Reply:
The intention of the `mode` check is that `copyTargetCount` can involve pagehits--I just merged a libram change that means it takes a lot fewer, but we still don't want to be checking it willy nilly